frontend/src/lib/components/Errors/ErrorDisplay.tsx

```diff
@@ -12,6 +12,20 @@ interface StackFrame {
     lineno: number
     colno: number
     function: string
+    context_line?: string
+}
+
+interface ExceptionTrace {
+    stacktrace: {
+        frames: StackFrame[]
+    }
+    mechanism: {
+        handled: boolean
```

Comment:
I'd change this to something else (e.g., String|enum) that accepts multiple values, such as `handled|unhandled|process_termination`. The reason is that Sentry was built for the backend and frontend, but different platforms have different needs.
more context here https://github.com/getsentry/rfcs/pull/10

Reply:
Ah nice, I think I'd rather just remove it for now, and think about how we want to handle it when its time to add it? It's unused anyway right now